inc/class-workflow.php

```diff
@@ -295,7 +295,7 @@ public function run( array $args = [] ) {
 						'paged' => -1,
 					] ) );
 				} elseif ( $this->event->get_recipient_handler( $recipient ) ) {
-					$results = call_user_func_array( $this->event->get_recipient_handler( $recipient ), $args );
+					$results = call_user_func_array( $this->event->get_recipient_handler( $recipient ), [ $args ] );
```

Comment:
Again, this would break the existing functionality that expects a list of function parameters, what you’re doing here is converting it to a single function parameter that’s an array.

as before though it would make sense to use array_values here:

```suggestion
					$results = call_user_func_array( $this->event->get_recipient_handler( $recipient ), array_values( $args ) );
```